spec/lib/vanagon/platform/windows_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'vanagon/platform'
+load File.expand_path(File.join(File.dirname(__FILE__), "../../../..", "lib", "vanagon.rb"))
```

Comment:
@haus Didn't appear to be loading, but in any case, for the purposes of the test, I need to redefine it anyway.